mm2src/coins/lightning/ln_conf.rs

```diff
@@ -129,10 +102,50 @@ pub struct OurChannelsConfig {
     /// our real on-chain channel UTXO in each invoice and requiring that our counterparty only
     /// relay HTLCs to us using the channel's SCID alias.
     pub negotiate_scid_privacy: Option<bool>,
+    /// Sets the percentage of the channel value we will cap the total value of outstanding inbound
+    /// HTLCs to.
+    pub max_inbound_in_flight_htlc_percent: Option<u8>,
+    /// Set to announce the channel publicly and notify all nodes that they can route via this
+    /// channel.
+    pub announced_channel: Option<bool>,
+    /// When set, we commit to an upfront shutdown_pubkey at channel open.
+    pub commit_upfront_shutdown_pubkey: Option<bool>,
```

Comment:
`OurChannelsConfigs` mirrors `ChannelHandshakeConfig` from the rust-lightning library, but it is deserializable to be used in requests and configs. https://github.com/KomodoPlatform/atomicDEX-API/blob/6d6ab4e0d3dc40bea6dbbeb3f5c795f7e265f8b3/mm2src/coins/lightning/ln_conf.rs#L88
Same for `ChannelOptions` and `CounterpartyLimits` as they mirror `ChannelConfig` and `ChannelHandshakeLimits` respectivally.

So all the fields in these structs are an `Option` because if they are not provided in the coin config the default value from the rust-lightning library is used instead, even if it's changed in new releases. These structs are also used in some requests. As an example https://github.com/KomodoPlatform/atomicDEX-API/blob/6d6ab4e0d3dc40bea6dbbeb3f5c795f7e265f8b3/mm2src/coins/lightning.rs#L835-L847 In `OpenChannelRequest` if `announced_channel` is not provided in `channel_options` for example, it means to use the value from config.